go.mod

```diff
@@ -0,0 +1,35 @@
+module code.cloudfoundry.org/dockerdriver
+
+go 1.19
+
+require (
+	code.cloudfoundry.org/cfhttp v2.0.0+incompatible
+	code.cloudfoundry.org/clock v1.1.0
+	code.cloudfoundry.org/goshims v0.14.0
+	code.cloudfoundry.org/lager/v3 v3.0.1
+	code.cloudfoundry.org/localdriver v0.0.0-20230411134712-bfbf40217301
```

Comment:
[localdriver also import dockerdriver](https://github.com/cloudfoundry/localdriver/blob/main/local_driver.go#L10). This circular dependency we have (inherited from days before go.mod existed) is one of the reasons why this repo has not been converted to it's own module. Essentially leading to a situation where there will always be a dependency that's out of date.

Reply:
Thank you @winkingturtle-vmw. We definitely don't want a circular dependency!

The dependency was only needed for a test. I've updated the test to grab the latest `localdriver` when it runs. This is not a perfect solution, but it better than a circular dependency.